src/components/Loadings/SkeletonLoading/SkeletonLoading.vue

```diff
@@ -0,0 +1,101 @@
+<template>
+  <div
+    class="items-container"
+    :style="secondaryPosition"
+  >
+    <div
+      class="main-item-content"
+      :style="mainContainerSize"
+    />
+    <div
+      class="secondary-item-content"
+      :style="secondaryContainerSize"
+    />
+  </div>
+</template>
+
+<script>
+export default {
+  name: 'SkeletonLoading',
+  props: {
+    mainContainerDimensions: {
+      type: Array,
+      default: () => [0, 0],
+    },
+    secondaryContainerDimensions: {
+      type: Array,
+      default: () => [0, 0],
+    },
```

Comment:
I think it would be better if these two props were objects containing 2 keys each (height, width)
```
{
  height: number
  width: number
}